### ISSUE TITLE
Investigate possible map circularity case in builder

### ISSUE THREAD

**Issue by guybedford** [systemjs/builder#585]
In the case - 

```javascript
builder.config({
        map: {
            'lodash': 'lodash/lodash.js',
            'spark-md5': 'spark-md5/spark-md5.js'
        },
        paths: {
            "app/": "./dist/public/app/"
        },
        packages: {
            '': {
                defaultExtension: 'js'
            },
```

Apparently this is throwing on `no such file.... node_modules/lodash/lodash.js/lodash.js`, even with no requires to `lodash/lodash.js` present at all in the codebase.

We never apply double maps, so it would be worth confirming where this is coming from.

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/builder/commit/3378663b56eed2940732166660e52fa8c0ff9251.

**Comment by guybedford** [comment]
Released in 0.15.17.
